edb/schema/indexes.py

```diff
@@ -125,6 +129,23 @@ class Index(
         inheritable=False,
     )
 
+    params = so.SchemaField(
```

Comment:
Add a comment around here making clear when `params` appears and when `kwargs` does?